plugins/transformations/sequence2array/srcCxx/Sequence2ArrayTransformation.cxx

```diff
@@ -291,15 +269,15 @@ bool Seq2ArrayTransformation::are_type_compatible(
     return is_compatible;
 }
 
-Seq2ArrayTransformation::Seq2ArrayTransformation(
+Sequence2ArrayTransformation::Sequence2ArrayTransformation(
         const rti::routing::TypeInfo &input_type_info,
         const rti::routing::TypeInfo &output_type_info,
         const rti::routing::PropertySet &properties)
         : input_type_info_(input_type_info.dynamic_type()),
           output_type_info_(output_type_info.dynamic_type())
 {
-    if (!are_type_compatible(input_type_info_, output_type_info_)) {
-        throw std::runtime_error("Error: input and ouput types are not compatible");
+    if (!are_types_compatible(input_type_info_, output_type_info_)) {
+        throw std::runtime_error("input and ouput types are not compatible");
```

Comment:
This exception will never be thrown at the moment, since `are_types_compatible()` will not return `false` but rather thrown an exception in case of incompatibility.

Reply:
changed